### ISSUE TITLE
ErrorException (E_NOTICE) Undefined index: function

### ISSUE THREAD

**Issue by Nuranto** [getsentry/sentry-laravel#244]
`vendor/sentry/sentry/src/Stacktrace.php` 

```            $functionName = sprintf('%s::%s', $backtraceFrame['class'], $backtraceFrame['function']);```

The (stupid) error (which I get with sentry disabled) was : 

```
 FatalErrorException (E_UNKNOWN)
Switch statements may only contain one default clause
```

Just a switch with 2 `default:`

**Comment by stayallive** [comment]
Hey @Nuranto could you show us a little more specific when this occurs and on what PHP version you are running we cannot seem to reproduce it on our side?

**Comment by Nuranto** [comment]
PHP 7.3.5
```
:./current# composer show | grep sentry
sentry/sdk                             2.0.3              This is a metapackage shipping sentry/sentry with a re...
sentry/sentry                          2.1.0              A PHP SDK for Sentry (http://sentry.io)
sentry/sentry-laravel                  1.0.2              Laravel SDK for Sentry (https://sentry.io)
```

This time, it happenned because I had two `default:` case in a switch. But it happenned already before on different errors. I will log it here next time it happens.

**Comment by stayallive** [comment]
Thanks! It will help if we have a reproducible case to see why this is happening!

**Comment by Nuranto** [comment]
Just tried to reproduce with double default in a switch in a controller, but I can't reproduce. So it is clearly not error-related.
I will try to give you more data.

**Comment by Nuranto** [comment]
Ok, I found a way to reproduce : 

```
<?php

namespace App;

class Toto {
    public function toto() {
        switch('toto') {
                default:
                default:
            }
    }
}
```
and inside a blade view : 
```
@php $toto = new \App\Toto(); @endphp
{{ $toto->toto() }}
```

**Comment by stayallive** [comment]
@Nuranto many thanks for giving us this case, it seems related to xdebug (which was the missing link in our tests).

We're tracking a fix here: getsentry/sentry#823, so I'll close this one.

Many thanks for giving us a reproducible case!
